### ISSUE TITLE
[Generic font families] "Slanted" category in Hebrew

### ISSUE THREAD

**Issue by iorsh** [w3c/i18n-discuss#32]
Commenting on https://github.com/w3c/i18n-discuss/wiki/Generic-font-families

I can't see any value for outstanding "slanted" category in Hebrew. Just like in Latin fonts, italic or slanted is used in a conjunction with roman type within the same family. Some Hebrew font families have oblique version which is just mechanical or nearly-mechanical slanting, while others pair a true italic version with an upright one.

The only difference is that in Latin the slanted/italic font is universally skewed to the right, while in Hebrew the skewing can be to the right or to the left according to designer's preference.

**Comment by r12a** [comment]
Thanks for the comment. 

Do you think there might be a call for directional slanting to be managed in generic font families?  CSS is already considering ways of stating such preferences for obliquing text (which is relevant for Arabic, and N'Ko scripts, too); ie. if the author wants the text to slant in the direction of reading, the browser would look for a font that slants text in the desired direction, rather than falling back to a random font which may slant to the right.  Maybe it's enough to handle that in the oblique settings??

Slanted text may be different in Hebrew from scripts like Tamil and Khmer, so i'll admit i wasn't sure about including it.  

I wonder whether we should recommend a generic category for Rashi fonts, since they tend to be used in specific situations, where you want the fallback to be resistant to random subsitutions(??).

**Comment by myoresh** [comment]
I think the directional slanting is already there, like {font-style: oblique 50deg;}, where the angular measure can be either positive or negative. So, on my option, there is no reason to deviate from Latin in the slanting / italic behavior for Hebrew.

The Rashi script does warrant a category of its own, provided we can guarantee that **some** Rashi font is present on the system. If there is no Rashi at all, I would still prefer to see Hebrew instead of rectangular boxes.

**Comment by iorsh** [comment]
I think the directional slanting is already there, like {font-style: oblique 50deg;}, where the angular measure can be either positive or negative. So, on my opinion, there is no reason to deviate from Latin in the slanting / italic behavior for Hebrew.

The Rashi script does warrant a category of its own, provided we can guarantee that **some** Rashi font is present on the system. If there is no Rashi at all, I would still prefer to see Hebrew instead of rectangular boxes.

**Comment by r12a** [comment]
> I think the directional slanting is already there, like {font-style: oblique 50deg;}, where the angular measure can be either positive or negative.

Yeah, but the browsers don't handle it well yet.  That's in discussion in the CSS WG.

> So, on my opinion, there is no reason to deviate from Latin in the slanting / italic behavior for Hebrew.

I'm inclined to agree.

> The Rashi script does warrant a category of its own, provided we can guarantee that some Rashi font is present on the system. If there is no Rashi at all, I would still prefer to see Hebrew instead of rectangular boxes.

Yes, the idea is that if no Rashi generic font is available, the browser will fall back to another generic (such as serif/sans-serif).

The only Rashi font i'm aware of provided by macOS, Windows11, Noto and SIL is a Noto font. See https://r12a.github.io/scripts/fontlist/index.html?script=hebr#other.

**Comment by r12a** [comment]
I have now changed both https://github.com/w3c/i18n-discuss/wiki/Generic-font-families and https://r12a.github.io/scripts/fontlist/ to add a `rashi` category.  I removed the reference to Hebrew `slanted` from several places, but it is partly intact still until i make a final decision as to whether that serves any useful purpose.  So i'll close this (at least for now). Feel free to add additional comments or reopen as needed.
